### PR TITLE
show search bar and job indicators on all screen sizes

### PR DESCRIPTION
**Pull request by lilytumblr**

### CODE REVIEW

File: app/views/layouts/_navigation.html.erb [hunk 1]
```diff
@@ -28,20 +28,20 @@
...
+      <form class="navbar-form navbar-left" role="search">
         <div class="form-group">
           <input type="text" class="form-control" id='agent-navigate' autocomplete="off" placeholder="Search">
           <%= image_tag "spinner-arrows.gif", :class => "spinner" %>
         </div>
       </form>
       
       <li class='job-indicator' role='pending'>
-        <%= link_to current_user.admin? ? jobs_path : '#', class: 'visible-lg' do %>
+        <%= link_to current_user.admin? ? jobs_path : '#', class: '.d-block' do %>
```
Comment (dsander) [inline app/views/layouts/_navigation.html.erb:39]: I don't think the class name `.d-block` is defined anywhere, can we get rid of `class: '.d-block'`?

Comment (lilytumblr) [reply app/views/layouts/_navigation.html.erb:39]: yes, but I dont know how to remove the class without causing something went wrong error, no knowledge of codes 😅can you modify it right?

Comment (dsander) [inline app/views/layouts/_navigation.html.erb:39]:
```suggestion
        <%= link_to current_user.admin? ? jobs_path : '#' do %>
```

File: app/views/layouts/_navigation.html.erb [hunk 2]
```diff
@@ -28,20 +28,20 @@
...
           <span class="badge"><%= icon_tag('glyphicon-refresh', class: 'icon-white') %> <span class='number'>0</span></span>
         <% end %>
       </li>
       <li class='job-indicator' role='awaiting_retry'>
-        <%= link_to current_user.admin? ? jobs_path : '#', class: 'visible-lg' do %>
+        <%= link_to current_user.admin? ? jobs_path : '#', class: '.d-block' do %>
```
Comment (dsander) [inline app/views/layouts/_navigation.html.erb:44]:
```suggestion
        <%= link_to current_user.admin? ? jobs_path : '#' do %>
```

Comment (lilytumblr) [reply app/views/layouts/_navigation.html.erb:44]: Thank you! Worked perfectly.😄

**Pull request merged**